extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -428,8 +428,8 @@ export class SchemaCompareMainWindow {
 						// show the first dependent that caused this to fail in the warning message
 						const diffEntryName = this.createName(diff.sourceValue ? diff.sourceValue : diff.targetValue);
 						const firstDependentName = this.createName(result.blockingDependencies[0].sourceValue ? result.blockingDependencies[0].sourceValue : result.blockingDependencies[0].targetValue);
-						const cannotExcludeMessage = localize('schemaCompare.cannotExcludeMessage', "Cannot exclude {0}. Included dependents exist such as {1}", diffEntryName, firstDependentName);
-						const cannotIncludeMessage = localize('schemaCompare.cannotIncludeMessage', "Cannot include {0}. Excluded dependents exist such as {1}", diffEntryName, firstDependentName);
+						const cannotExcludeMessage = localize('schemaCompare.cannotExcludeMessage', "Cannot exclude {0}. Included dependents exist {1}", diffEntryName, firstDependentName ? 'such as ' + firstDependentName : '');
```

Comment:
Since this is being used for both messages, can this be defined as a const and reused like const existingDependentsMessage = firstDependentName ? `such as ${firstDependentName}` : '';

Reply:
This isn't correct regardless - the `such as` will never be localized as it is currently. Only the second string parameter is localized - the others are added in as-is. I would suggest two separate messages - one with just `Included dependents exist.` and the other with `Included dependents exist, such as {1}`.

Reply:
Yeah I would suggest having two separate strings for both cases, appending strings (even two loc strings) might not mean the same as we indent in all languages.

My next request - when we dont have the name - can we atleast shre type information... something like - "Included <table> dependents exist". @kisantia  any insights here?

Reply:
Yes it would be good to put the object type which is DiffEntry.name

Reply:
As discussed offline, skipped update to add object type. And updated the message to be separate.